toolset/setup/linux/prerequisites.sh

```diff
@@ -42,7 +42,8 @@ sudo apt-get -qqy install -o Dpkg::Options::="--force-confdef" -o Dpkg::Options:
   libpq-dev mlton \
   libjemalloc-dev libluajit-5.1-dev `# Needed by lwan at least` \
   libhiredis-dev                    `# Redis client - Needed by ngx_mruby at least` \
-  cloc dstat                        `# Collect resource usage statistics`
+  cloc dstat                        `# Collect resource usage statistics` \
+  libsasl2-dev                      `# Needed by mgo for go-mongodb test`
```

Comment:
Everything looks good but this - is there any way to avoid putting yet another test-specific dependency into `prerequisites.sh` - especially one that requires `sudo`?